lib/ex_trello/api/boards.ex

```diff
@@ -49,7 +49,6 @@ defmodule ExTrello.API.Boards do
 
   # Board labels
   def labels(board), do: labels(board, [])
-  def labels(%Board{id: id}, options), do: labels(id, options)
```

Comment:
This isn't unnecessary. It's entirely consistent with many other functions in this very module that allow passing the struct for which you're finding some sub-resource for(instead of only the id).